08-projects.Rmd

```diff
@@ -6147,6 +6172,8 @@ A public sample of the data can be found [here](https://www.datadepot.rcac.purdu
 
 ##### 2. Examine the `episodes` table. Based on observation and the column names, do you think there are any primary keys? How about foreign keys?
 
+**Hint:** A primary key can also be a foreign key.
```

Comment:
Hi @kevinamstutz , I add the hint here simply because I forgot that `episode_title_id` is also a foreign key! However, if you think this would make the question too straight-forward, feel free to remove the hint.

Reply:
Hey @ymzhang-neo, I agree with the addition.